pyspeckit/spectrum/tests/test_units.py

```diff
@@ -9,6 +9,28 @@
                   for b in units.unit_type_dict if b not in ('unknown',None)
                   for c in convention
                   for d in ['GHz','cm']]
+params = params[:5]
+
+
+def test_equivalencies_1():
+    x = SpectroscopicAxis(np.arange(5), unit=u.angstrom, equivalencies=u.doppler_optical(3*u.AA))
+    assert x.equivalencies == u.doppler_optical(3*u.AA)
+
+@pytest.mark.parametrize(('velocity_convention'), convention)
+def test_equivalencies_2(velocity_convention):
+    x = SpectroscopicAxis(np.arange(5), unit=u.angstrom, refX=3*u.AA, velocity_convention=velocity_convention)
+    assert x.equivalencies == u.doppler_optical(3*u.AA)
```

Comment:
This is incorrect: if you're parametrizing over velocity convention, i.e. using `convention='optical'` in one case and `convention='radio'` in another case, the equivalencies should not be `doppler_optical` in both cases.

I also suspect that this test won't actually work after all because this:
```
u.doppler_optical(5*u.AA) == u.doppler_optical(5*u.AA)
```
is False!

We need to come up with a cleverer way to test whether the equivalencies are correct.  Perhaps it would be better to check if the `refX` is appropriately set... I will mention this below.